src/components/AccountForms/RegistrationForm/RegistrationForm.test.js

```diff
@@ -19,26 +19,53 @@ describe('<RegistrationForm />', () => {
     );
   });
 
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should check that titles and mentor and mentee buttons render to page', async () => {
+    await act(async () => {
+      const formTitle = screen.getByText('Create account');
+      expect(formTitle).toBeInTheDocument();
+
+      const formSubtitle = screen.getByText('Have an existing account?');
+      expect(formSubtitle).toBeInTheDocument();
+
+      const loginLink = screen.getByText('Login');
+      expect(loginLink).toBeInTheDocument();
+
+      const chipSelectText = screen.getByText('I want to sign up as a:');
+      expect(chipSelectText).toBeInTheDocument();
+
+      const mentorSelect = screen.getByText('Mentor');
+      expect(mentorSelect).toBeInTheDocument();
+      fireEvent.click(mentorSelect);
+
```

Comment:
good test. we can do more by mocking to see if the button was clicked.

Reply:
Thanks Jackson. I test that the button is clicked in the test below this one where it checks that the submit handler fires(line 52). Is that what you mean or should I do more?

Reply:
Ah that is fine. 👍